### ISSUE TITLE
Misc bugs from Trainees

### ISSUE THREAD

**Issue by JoannaRumbley** [attendanceproject/djattendance#1373]
**Submitting Personal Attendance:**
A couple trainees seem to have trouble submitting their attendance. (Ady Garcia showed me hers) When they enter their rolls and click Submit, it gives them an error that says something like, "please select all your rolls before finalizing". (I don't remember the exact message) Not sure what the issue is exactly...

(From Kristie Ahn) 

- [ ] Loading personal attendance page is slow (she talked with @zioninzion about it. The issue was fixed then came back this morning 8/21)

- [ ] Absent trainee roster says "submitted" before it was submitted

**Comment by iamsitting** [comment]
For personal attendance: They finalized accidentally.

**Comment by JoannaRumbley** [comment]
I don't believe that's the issue. Mine already says "Please submit rolls before finalizing." And I haven't clicked anything (cuz I'm currently not on self-attendance). Either way, they shouldn't need to mark themselves Present for the rolls they are present for. The default is already Present, right? (that's what the AMs told everyone) If someone has perfect attendance, then this message doesn't make sense. 

But if they did accidentally click Finalize, then the Finalize button needs to be grayed out so they can't click on it twice. They also shouldn't be able to click Finalize until the end of the week. If it's already like this, maybe there should be some text saying "You have already finalized this week" or something.

**Comment by iamsitting** [comment]
Yes, the message is unclear. It needs to be reworded.
The message is there all the time whether you finalize or not. The
There's no need to enter rolls to finalize. Including for perfect attendance.
The finalize button is made unclickable if it's clicked.

**Comment by JoannaRumbley** [comment]
Ok thanks for clarifying that!
